internal/eval/eval_test.go

```diff
@@ -2103,12 +2182,181 @@ func testEvalPFCOUNT(t *testing.T, store *dstore.Store) {
 				}
 				store.Put(key, obj)
 			},
-			input:  []string{"EXISTING_KEY"},
-			output: clientio.Encode(1, false),
+			input: []string{"EXISTING_KEY"},
+			migratedOutput: EvalResponse{
+				Result: uint64(1),
+				Error:  nil,
+			},
 		},
 	}
 
-	runEvalTests(t, tests, evalPFCOUNT, store)
+	for _, tt := range tests {
```

Comment:
This part seems to be same for all 3 tests. May be combine them into one function and call that function here. Or you can also take a look at `runEvalTests` or `runMigratedEvalTests` function and see if they can be reused here ?

Reply:
I could have written it as a single function and reuse it, but again, there are so many people working on migrations and if there are multiple of those functions with different names, they would all be redundant. Since you are suggesting to use `runMigratedEvalTests` I will go with that. Thanks for the suggestion